ui/pages/swaps/smart-transaction-status/smart-transaction-status.js

```diff
@@ -441,7 +451,8 @@ export default function SmartTransactionStatusPage() {
         {description && (
           <Text
             variant={TextVariant.bodySm}
-            boxProps={{ ...(blockExplorerUrl && { margin: [1, 0, 0] }) }}
+            as="h6"
+            marginTop={1}
```

Comment:
Sorry I missed the conditional
```suggestion
            marginTop={blockExplorerUrl && 1}
```